src/uct/cuda/base/cuda_md.c

```diff
@@ -262,6 +262,11 @@ ucs_status_t uct_cuda_base_mem_query(uct_md_h tl_md, const void *address,
         mem_attr->alloc_length = addr_mem_info.alloc_length;
     }
 
+    if (mem_attr->field_mask & UCT_MD_MEM_ATTR_FIELD_DMABUF_FD) {
+        /* unsupported */
+        mem_attr->dmabuf_fd = UCT_DMABUF_FD_INVALID;
```

Comment:
i think we should  add DMABUF capability to md_query

Reply:
@yosefe any specific advantage in extending MD capabilities to notify whether MD can return dmabuf for a given memory region? As dmabuf is associated with a given memory block, isn't it a better fit to query with md_mem_query for the memory block of interest and see if the given MD can provide a dmabuf_fd? If there's an advantage in extending MD capabilities, I can add that as well.